bin/generate-test-app.sh

```diff
@@ -8,9 +9,9 @@ mkdir rails-test-app/spec/models
 mkdir rails-test-app/spec/system
 mkdir rails-test-app/spec/requests
 
-echo "Note! CI is not allowing our build scripts to write to package.json"
-echo "Note! This package.json may need to be updated in the future."
-cp test-app/package.json rails-test-app/package.json
+# echo "Note! CI is not allowing our build scripts to write to package.json"
+# echo "Note! This package.json may need to be updated in the future."
+# cp test-app/package.json rails-test-app/package.json
```

Comment:
Want to keep these comments or remove them?

Reply:
These get un-commented later on.